src/app/settings/organization-plans.component.html

```diff
@@ -165,6 +165,13 @@ <h2 class="spaced-header">{{'summary' | i18n}}</h2>
                         {{selectablePlan.basePrice | currency:'$'}}
                         /{{'year' | i18n}}
                     </small>
+                    <small *ngIf="acceptingSponsorship">
```

Comment:
It looks like you're missing this bool flag on the above `ngIf` as well

Reply:
I did have it there but it's not needed as the basePrice for the sponsorship is 0. Therefore it's already `falsey` and doesn't display. If you want it there for verbosity though and I can add it.

Reply:
Added it above as well since we are no longer changing the `basePrice` to 0 and we need it isn't `falsey`.